stable/node-problem-detector/README.md

```diff
@@ -1,42 +1,16 @@
 # node-problem-detector
 
-![Version: 2.3.1](https://img.shields.io/badge/Version-2.3.1-informational?style=flat-square) ![AppVersion: v0.8.12](https://img.shields.io/badge/AppVersion-v0.8.12-informational?style=flat-square)
+![Version: 2.3.2](https://img.shields.io/badge/Version-2.3.2-informational?style=flat-square) ![AppVersion: v0.8.12](https://img.shields.io/badge/AppVersion-v0.8.12-informational?style=flat-square)
 
 This chart installs a [node-problem-detector](https://github.com/kubernetes/node-problem-detector) daemonset. This tool aims to make various node problems visible to the upstream layers in cluster management stack. It is a daemon which runs on each node, detects node problems and reports them to apiserver.
 
 **Homepage:** <https://github.com/kubernetes/node-problem-detector>
 
-## How to install this chart
```

Comment:
You can't remove all this 🙂

Reply:
I did not :-) The helm-docs container run did this automatically. Not sure if there is something missing in the template file for `helm-docs`?

Reply:
I think you're not running it from the repo root as per the instructions?

Reply:
Running this in the root dir of the repo clearly helps 🤦  Sorry for the confusion...